test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature

```diff
@@ -1031,23 +1032,33 @@ Feature: get an attribute value by entity ID and attribute name if it exists usi
       | house_&             |
       | my house            |
 
+  @attribute_name_invalid @BUG_1887
+  Scenario:  try to get an attribute value by entity ID and attribute name using NGSI v2 with invalid attribute_name
+    Given  a definition of headers
+      | parameter          | value                 |
+      | Fiware-Service     | test_update_entity_id |
+      | Fiware-ServicePath | /test                 |
+      | Accept             | text/plain            |
+    When get an attribute value by ID "room_1" and attribute name "house_/" if it exists
+    Then verify that receive an "Bad Request" http code
+    And verify an error response
+      | parameter   | value             |
+      | error       | BadRequest        |
+      | description | service not found |
+
   @attribute_name_invalid @BUG_1887 @skip
```

Comment:
yes, it is scenario and another with invalid id using `house_#` return an 406 http code. Commented in the issue #1887